tests/index.test.js

```diff
@@ -28,6 +33,17 @@ icons.forEach(icon => {
 
   test(`${icon.title} has an "svg" value`, () => {
     expect(typeof subject.svg).toBe('string');
+    const svgFileContents = fs.readFileSync(svgPath, 'utf8')
+      .replace('" />', '"/>')
+      .replace(/\r?\n/, '')
+      .replace('></path>', '/>')
+      .replace(
+        'xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\">',
+        'viewBox=\"0 0 24 24\" xmlns=\"http://www.w3.org/2000/svg\">')
+      .replace(
+        /<title>([^<]+)<\/title>/,
+        `<title>${titleToHtmlFriendly(subject.title)}</title>`);
```

Comment:
From the Jest output you shared:

```
[...]
Expected: [...]<title>Let&apos;s Encrypt</title>[...]
Received: [...]<title>Let&amp;apos;s Encrypt</title>[...]
[...]
```

Looks like there's bug in `titleToHtmlFriendly` (ampersand should be replaced first), good thing we tested it this way :upside_down_face:

Reply:
It is a problem of unifying the data. We want that these `title`s must be html-encoded in _icons/*.svg_ SVG contents or rather we should accept the names html-decoded in _icons/*.svg_ and encode them in the `svg` getter building the package?

Reply:
I think we should accept only encoded names and fix the ones that are not encoded.

Reply:
> I think we should accept only encoded names and fix the ones that are not encoded.

I agree and suggest to create another lint rule for this. What do you think @ericcornelissen?

Reply:
I'm not sure if I follow @mondeja. Here's what I can say:

- The `titleToHtmlFriendly` currently in this Pull Request is incorrect, as can be seen from the "Let’s encrypt" example ("Let’s encrypt" -> "Let&apos;s encrypt" -> "Let&amp;apos;s encrypt"). This should definitely be fixed.
- The `.svg` file _should_ all already have properly encoded `<title>` values. If not, that should be fixed. If we can lint for that, we probably should.
- I think the `svg` value in all `js` files should be an exact match of the contents of the corresponding SVG so that it can be embedded in HTML as is. This is the way it has always been, and changing that could definitely be considered a breaking change from my point of view. If you want to add a non-HTML safe version of the SVG in the JavaScript (whether hardcoded or through a helper function), I guess we can do that but I would have no clue as to why anyone would want that.

Reply:
Opened a PR for this in #6522